### ISSUE TITLE
Proportion plotting

### ISSUE THREAD

**Issue by Marius1311** [theislab/scvelo#243]
I'm trying to run `scv.pl.proportions(adata, groupby='leiden')`, where leiden is the standard categorical output of `sc.tl.leiden`. I'm getting

```pytb
---------------------------------------------------------------------------
IndexError                                Traceback (most recent call last)
<ipython-input-92-7462a143685b> in <module>
----> 1 scv.pl.proportions(adata, groupby='leiden')

~/Projects/scvelo/scvelo/plotting/proportions.py in proportions(adata, groupby, layers, highlight, add_labels_pie, add_labels_bar, fontsize, figsize, dpi, use_raw, show)
     76     autopct = "%1.0f%%" if add_labels_pie else None
     77     pie = ax.pie(
---> 78         np.mean(counts_layers, axis=1),
     79         colors=colors,
     80         explode=explode,

<__array_function__ internals> in mean(*args, **kwargs)

~/miniconda3/envs/cellrank/lib/python3.6/site-packages/numpy/core/fromnumeric.py in mean(a, axis, dtype, out, keepdims)
   3333 
   3334     return _methods._mean(a, axis=axis, dtype=dtype,
-> 3335                           out=out, **kwargs)
   3336 
   3337 

~/miniconda3/envs/cellrank/lib/python3.6/site-packages/numpy/core/_methods.py in _mean(a, axis, dtype, out, keepdims)
    136 
    137     is_float16_result = False
--> 138     rcount = _count_reduce_items(arr, axis)
    139     # Make this warning show up first
    140     if rcount == 0:

~/miniconda3/envs/cellrank/lib/python3.6/site-packages/numpy/core/_methods.py in _count_reduce_items(arr, axis)
     55     items = 1
     56     for ax in axis:
---> 57         items *= arr.shape[ax]
     58     return items
     59 

IndexError: tuple index out of range
```

#### Versions:
scvelo==0.2.2.dev51+ga7de78a  scanpy==1.5.1  anndata==0.7.3  loompy==3.0.6  numpy==1.18.5  scipy==1.4.1  matplotlib==3.1.2  sklearn==0.22  pandas==1.0.5  
>

**Comment by WeilerP** [comment]
@Marius1311, sorry for picking this up only now. Is this still an open problem? If so, is the error thrown using the latest version of `scvelo` and `scanpy`? If you managed to solve it, would you mind elaborating on what caused the error?

**Comment by Marius1311** [comment]
let me check whether this still persists

**Comment by Marius1311** [comment]
I can confirm that this has been solved in scVelo `0.2.3`! Thank's for taking this up again, good initiatieve to close these old issues!
